tests/integration/drivers/vector/test_astra_db_vector_store_driver.py

```diff
@@ -14,7 +14,7 @@
 
 
 class TestAstraDbVectorStoreDriver:
-    @pytest.fixture()
+    @pytest.fixture
```

Comment:
Why reverting this change? Did we change formatting rules?

(looks like this is the biggest diff contributor in the pr)

Reply:
Ah I just realized that the original is the ["correct"](https://docs.astral.sh/ruff/rules/pytest-fixture-incorrect-parentheses-style/) way. I had it backwards, will investigate and revert.